depends/hosts/darwin.mk

```diff
@@ -40,19 +48,14 @@ llvm_config_prog=$(shell $(SHELL) $(.SHELLFLAGS) "command -v llvm-config")
 
 llvm_lib_dir=$(shell $(llvm_config_prog) --libdir)
 
+darwin_AR=$(shell $(SHELL) $(.SHELLFLAGS) "command -v llvm-ar")
+darwin_DSYMUTIL=$(shell $(SHELL) $(.SHELLFLAGS) "command -v dsymutil")
```

Comment:
Needs update?

Reply:
In Guix this should just be dsymutil, not llvm-dsymutil.

Reply:
Clang actually expects this all the time, so just dsymutil is correct in depends as well (so we can't copy the bin as llvm-dsymutil) (this was the qt related issue I had seen before).